app/views/posts/_posts.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= link_to(user_post_path(@user, post)) do %>
+
+  <div class="post-container"> 
+    <div class="post">
+      <h3><%= post.title %></h3>
+      <p><%= post.text[0..25] %> ...</p>
+    </div>
+    <div class="actions d-flex">
+      <span>Comments: <%= post.comments_counter%></span>
+      <span>Likes: <%= post.likes_counter%></span>
+    </div>
+  </div>
+
+  <div class="comments">
+    <% if post.comments == [] %>
+      <p>No comments !!!</p>
+    <% end %>
+  
+    <% post.comments.order('created_at DESC').limit(5).each do |comment|%>
+      <div class="comment-container">
+        <span><%=comment.author.name %>:</span>
+        <span><%=comment.text %></span>
+      </div>
+    <% end %>
```

Comment:
- [ ] Good job here 👏 👏 However we are expected to use the custom method we created back in the **post model** which retrieves most recent 5 comments on  a particular posts 😃 So please use it in order to render comments on each post 
HINT:  It might look like this 👇 
```erb
 <% post.most_recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```